src/app/group/GroupList.js

```diff
@@ -48,7 +49,7 @@ const GroupList = () => {
                                   tableDataStyle={{textAlign: "left"}}/>
 
                     <CreateGroupModal show={showCreateGroupModal}
-                                      onClose={() => setShowCreateGroupModal(false)}
+                                      onModalClosed={() => setShowCreateGroupModal(false)}
```

Comment:
I think it should be `onModalClose`, based on `onSomethingClick` rather than `onSomethingClicked`.